torch/csrc/jit/tensorexpr/codegen.h

```diff
@@ -49,6 +49,10 @@ class TORCH_API CodeGen {
     return device_;
   }
 
+  virtual std::string getCodeText() {
```

Comment:
What does it do?

Reply:
Glow needed to get generated code from NNC (in this case Block codegen), which it would then compile. This was a hook for that.  Glow had an access to codegen but not the exact type of codegen.

Reply:
I see, makes sense. Could you please add a comment for this function?

Reply:
In a longer term I think it probably should be generalized (like, it could return `void*`, for instance, and give some representation of the code produced by the codegen - it could be a string, but doesn't have to in a general case). Returning a pointer to the codegen itself also might be a viable option I think.

Reply:
That makes a lot of sense! I'll look into this

Reply:
Right now for the current pipeline we did return the pointer to the codegen.

Reply:
I should also rename this function. Will add that in the TODO.